docs/deep-learning/fhe_assistant.md

```diff
@@ -77,7 +77,7 @@ concrete_clf.compile(X, debug_config)
 
 #### 3. Quantization import failed
 
-**Error message**: `Error occurred during quantization aware training (QAT) import [...] Could not determine a unique scale for the quantization!`.
+**Error message**: `Error occurred during quantization aware training (QAT) import [...] Are you missing a QuantIdentity layer in your Brevitas model?`.
```

Comment:
We also have a non-intuitive  error that occurs when using 'view' instead of 'reshape'.

Error: ValueError: matmul: Input operand 1 has a mismatch in its core dimension 0, with gufunc signature (n?,k),(k,m?)->(n?,m?) (size ... is different from ...)

Maybe we could add it ? CC @andrei-stoian-zama

Reply:
that's not related, it's due to shape inference during the onnx export.